.github/workflows/ci.yml

```diff
@@ -1,14 +1,34 @@
 name: Build
-on: [push]
+on:
+  - push
 jobs:
-  build-test:
+  lint:
     runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.2.2
+          bundler-cache: true
+      - name: Lint
+        run: |
+          gem install rubocop -v 1.50.2
+          gem install rubocop-rspec -v 2.20.0
+          rubocop
 
+  build-test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - '2.5'
+          - '3.2.2'
```

Comment:
On CI, we are now running:

- lint, on Ruby 3.2.2 (newest)
- tests on:
    - 2.5 – oldest we can support based on syntax
    - 3.2.2 – newest version of Ruby